CrystalElseyNSS/aquarium-directory
# organized files, edited nav links, positioned tips section

### Changes 
- organized files in VSC
- amended nav and file links so they take user to correct locations on site
- positioned tips secion

### Testing
- make sure all links work
- make sure travel tips are at the top of locations page and flex correctly
